### PR TITLE
chore: bump kommander chart version to 0.8.0

### PR DESCRIPTION
**Pull request by makkes**

### CODE REVIEW

File: addons/kommander/1.1.x/kommander.yaml [hunk 1]
```diff
@@ -6,8 +6,8 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: kommander
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.1.0-18"
-    appversion.kubeaddons.mesosphere.io/kommander: "1.1.0-rc.1"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.1.0-19"
+    appversion.kubeaddons.mesosphere.io/kommander: "1.1.0-rc.2"
```
Comment (juliangieseke) [inline addons/kommander/1.1.x/kommander.yaml:10]:
```suggestion
```
… dont just increase all numbers that show up 🙈 😅 its still rc.1

Comment (makkes) [reply addons/kommander/1.1.x/kommander.yaml:10]: I don't understand that. We're obviously changing functionality. Why should the version number stay the same?

Comment (makkes) [reply addons/kommander/1.1.x/kommander.yaml:10]: I would actually suppose we **always** have to increase both version numbers whenever we bump the chart.

Comment (juliangieseke) [reply addons/kommander/1.1.x/kommander.yaml:10]: kommander does not release on every merge to master, that version is going to be the next release we do.

**Pull request closed without merging**